apis/config.go

```diff
@@ -58,7 +58,11 @@ type EtcdAdmConfig struct {
 	CacheDir   string
 	PodSpecDir string
 
-	BindAddr string
+	BindAddr   string
+	ClientPort int
+	PeerPort   int
+
+	CertValid int
```

Comment:
`CertValidity time.Duration` or `CertLifetime time.Duration` might be clearer here - I originally thought that we were turning off cert validation, and also by using time.Duration we avoid having to know that it is in years.